airflow-core/src/airflow/ui/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -16,23 +16,56 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { Box, Heading } from "@chakra-ui/react";
+import { Box, Heading, VStack } from "@chakra-ui/react";
+
+import type { UIAlert } from "openapi/requests/types.gen";
+import ReactMarkdown from "src/components/ReactMarkdown";
+import { Accordion, Alert } from "src/components/ui";
+import { useConfig } from "src/queries/useConfig";
 
 import { Health } from "./Health";
 import { HistoricalMetrics } from "./HistoricalMetrics";
 import { Stats } from "./Stats";
 
-export const Dashboard = () => (
-  <Box>
-    <Heading mb={4}>Welcome</Heading>
+export const Dashboard = () => {
+  const confAlerts = useConfig("dashboard_alert");
+  const alerts = Array.isArray(confAlerts) ? confAlerts : [];
```

Comment:
Nit" `Array.isArray(confAlerts)` why this check ? The backend seem to only be able to return a list there. What are we trying to handle ?

Reply:
There's an issue with the type checking on useConfig. It is ok to use `as Array<UIAlert>` here to make the type checking happy for now